app/views/paye/RtiDisabledHistoricPayAsYouEarnView.scala.html

```diff
@@ -135,7 +135,7 @@ <h3 class="govuk-heading-l flush break-word-only">@employment.name</h3>
                         @link(
                             url=routes.YourTaxCodeController.prevTaxCodes(viewModel.taxYear).toString(),
                             copy=messages("tai.taxCode.description.link"),
-                            linkClasses=Seq("govuk-link govuk-body-m")
+                            linkClasses=Seq("govuk-link govuk-body-m rtitaxcode")
```

Comment:
just out of curiosity why was this done here?

Reply:
existing  test fails because we are checking for the presence of  a tax code link by a generic class  (govuk-link)   - the UR banner also has a link with the same class (govuk-link)  therefore  to distinguish it added a specific empty class